kubernetes-manifests/ledger-db.yaml

```diff
@@ -103,6 +103,11 @@ spec:
             - mountPath: /var/lib/postgresql/data
               name: postgresdb
               subPath: postgres
+          securityContext:
```

Comment:
That's a start but it's also missing security contexts to prevent all access to root:

```
        securityContext:
          allowPrivilegeEscalation: false
          capabilities:
            drop:
              - all
          privileged: false
          readOnlyRootFilesystem: true
```

Reply:
I have added these now to the `ledger-db.yaml` file.